doc/source/security.md

```diff
@@ -4,7 +4,64 @@ The information in this document focuses primarily on cloud based deployments. F
 
 Brad Geesamen gave a wonderful talk titled [Hacking and Hardening Kubernetes by Example](https://kccncna17.sched.com/event/CU6z/hacking-and-hardening-kubernetes-clusters-by-example-i-brad-geesaman-symantec) at Kubecon NA 2017. You can [watch the talk](https://www.youtube.com/watch?v=vTgQLzeBfRU) or [read the slides](https://schd.ws/hosted_files/kccncna17/47/Hacking%20and%20Hardening%20Kubernetes%20By%20Example%20v1.pdf). Highly recommended that you do so to understand the security issues you are up against when using Kubernetes to run JupyterHub.
 
-## Secure the access to Helm
+## HTTPS
+
+Enabling HTTPS for your hub is pretty easy, since we automatically integrate with [Let's Encrypt](https://letsencrypt.org/).
```

Comment:
I really like the times when you've added links to a blog post etc in motivating the various topics in z2jh (e.g., like in the recent security PR). Can you think of something we could link to from here for users that may not have an intuition for why they'd use HTTPS? (not blocking on this PR but if one comes to mind I think it'd be helpful!)